### DISCUSSION TITLE
Nucleation of a single seed (benchmark problem)_ Unable to get seed to grow

### DISCUSSION THREAD

**Discussion by StephanieSharay22** [idaholab/moose#24711] [Q&A Modules: Phase field] [answered]
Hello all! 

I have been working on some of the phase field benchmark problems for Moose. For nucleation of a single seed, I have been unsuccessful in getting my single seed nucleation to grow in size at and above the critical radius value. A link of the problem statement (part a) is listed below. In addition, a zip of my input on csv files are attached. 

Thank you in advance for any help!

https://pages.nist.gov/pfhub/benchmarks/benchmark8.ipynb/

Files : https://github.com/StephanieSharay22/Nucleation_SingleSeed
[nucleation_single2.zip](https://github.com/idaholab/moose/files/11763318/nucleation_single2.zip)

**Comment by GiudGiud** [comment]
Hello

I wont be able to help here, but others may. @recuero @amjokisaari 
I would advise you create a github repository with your inputs rather than share a zip file. Most people will not download zip files from the internet. 

Guillaume

**Reply by StephanieSharay22** [reply]
Hi there! I went ahead and created a repository for the files. Thanks for the advice!

**Comment by liang-tech** [comment]
hello, from your input file, I don not find the [DiscreteNucleationInserter](https://mooseframework.inl.gov/source/userobjects/DiscreteNucleationInserter.html) and [DiscreteNucleationMap](https://mooseframework.inl.gov/source/userobjects/DiscreteNucleationMap.html) in the [UserObjects]. How do you insert a nuclei?

**Reply by StephanieSharay22** [reply]
Hello! It is inserted with the smoothcircleIC. The equation built in to it corresponds to the critical radius. The exact equation is equation 16 in the description link. Thank you!

**Comment by GiudGiud** [comment]
Any update here?
@laagesen have we had someone work on these benchmarks?

**Reply by StephanieSharay22** [reply]
Not quite! Closer though, I believe. Will be sure to update the input when we do!

**Comment by laagesen** [comment]
Hello! These have been solved with MOOSE, although the person who did this originally was a postdoc at Argonne and I don't believe he is there anymore. However someone else from our team may have looked at a similar problem, I will check on this.

**Comment by laagesen** [comment]
@sourabhkadambi has worked on this in the past and should be able to help. He also generated the nucleation section of this best practices page using MOOSE, which may be useful: https://github.com/usnistgov/pfhub/wiki/Problem-Setup

**Reply by StephanieSharay22** [reply]
Thank you!

**Comment by sourabhkadambi** [comment]
Hello! The problem is sensitive to the diffuse interface width, so it's important to set the required profile type and width with  SmoothCircleIC (see documentation at https://mooseframework.inl.gov/source/ics/SmoothCircleIC.html). The interface width is defined as the spatial distance between some choice of order parameter limits, e.g. 0.1 and 0.9. Eq. 4 in https://pages.nist.gov/pfhub/benchmarks/benchmark8.ipynb/ is specific to the tanh profile, but I'm not sure what limits they used to derive the expression. The definition `2.2 \sqrt(2) *\sqrt(kappa/W)` seems to work better. You can try setting `int_width = 3.1113` and `profile = TANH` in your SmoothCircleIC block.

**Reply by StephanieSharay22** [reply]
Hi there! Thank you so much for your answer! I tried setting my int_width = 3.1113 and profile = TANH, but unfortunately, the seed still will not grow and my total free energy continues to increase. I'll continue working through the problem. Thanks for all the help!!

**Reply by sourabhkadambi** [reply, marked as answer]
Please define the material property names in your `DerivativeParsedMaterial` blocks following https://mooseframework.inl.gov/source/materials/DerivativeParsedMaterial.html, i.e. `material_property_names = 'g(eta) p(eta)'`. If you don't declare the variable dependence for the materials, the derivates needed for the driving force for evolution may not be available. Someone else can confirm if the situation is the same with `ADDerivativeParsedMaterial`. Also, set a smaller time step; `dt = 0.5` worked for me. Let us know if this worked.

**Reply by StephanieSharay22** [reply]
It worked! Thank you so much!
